src/frequenz/repo/config/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 The tools are provided to configure the main types of repositories most commonly used at
 Frequenz, defined in
-[`freq.repo.config.RepositoryType`][freq.repo.config.RepositoryType].
+[`frequenz.repo.config.RepositoryType`][].
```

Comment:
the text side is empty,  would the mkdoc just use the package name?

Reply:
Yes, exactly, is a shortcut if you use a FQN you can omit the target.